### PR TITLE
1_8 Documentation

### PR DESCRIPTION
**Pull request by ErhanKRL**

### CODE REVIEW

File: src/components/questionComponent.js [hunk 1]
```diff
@@ -1,14 +1,13 @@
...
-import { NEXT_QUESTION_BUTTON_ID } from '../constants.js';
-
 /**
  * Create a full question element
- * @returns {Element}
+ * @param {string} question - The text of the question.
+ * @returns {HTMLDivElement} - The full question element
  */
+import { ANSWERS_LIST_ID } from '../constants.js';
+import { NEXT_QUESTION_BUTTON_ID } from '../constants.js';
```
Comment (NouranMahmoud) [inline src/components/questionComponent.js]: Imports always should be set in the very top of every file!

Comment (ErhanKRL) [reply src/components/questionComponent.js]: updated

**Review by Tenzijn — APPROVED** [review]

File: docs.md
Comment (NouranMahmoud) [inline docs.md:1]: Generate the docs please

File: src/components/answerComponent.js [hunk 2]
```diff
@@ -1,19 +1,21 @@
-import { checkAnswer } from '../utils/checkAnswer';
-
+/**
+ * @description Creates a li element for each answer
```
Comment (NouranMahmoud) [inline src/components/answerComponent.js:2]: You need to add a title for every function

**Pull request closed without merging**